action.yml

```diff
@@ -152,20 +156,26 @@ runs:
         fi
 
         # Determine command based on event type and PR status
+        max_turns="${{ inputs.max-turns }}"
```

Comment:
Can we rename this to $kodelet_args so going forward all the args will be added to this var